builders/interact/ethereum-api/libraries/ethersjs.md

```diff
@@ -5,11 +5,16 @@ description: Learn how to use the Ethereum EtherJS Library to send transactions
 
 # Ethers.js JavaScript Library
 
+<style>.video-container{display:flex;justify-content:space-between;max-width:1000px;margin:0 auto;}.video-column{width:49%;}.embed-container{position:relative;padding-bottom:56.25%;height:0;overflow:hidden;max-width:100%;margin-bottom:10px;}.embed-container iframe,.embed-container object,.embed-container embed{position:absolute;top:0;left:0;width:100%;height:100%;}.caption{font-family:Open Sans,sans-serif;font-size:0.9em;color:rgba(170,170,170,1);font-style:italic;letter-spacing:0px;position:relative;}</style><div class="video-container"><div class="video-column"><div class='embed-container'><iframe src='https://www.youtube.com/embed/bXtknNifO78?si=Hjiah5yhKvBcsE7f' frameborder='0' allowfullscreen></iframe></div><div class="caption">How to Send a Simple Transaction</div></div><div class="video-column"><div class='embed-container'><iframe src='https://www.youtube.com/embed/m9iVeCP7owI?si=krm6z9AsFCDrjlJh' frameborder='0' allowfullscreen></iframe></div><div class="caption">How to Deploy a Contract</div></div></div>
```

Comment:
I'm sorry for the back and forth! I just re-read Alberto's last comment about the captions, and I agree. I don't think we need the caption or text underneath the videos, but I'm also not opposed to them. So, I would agree with Alberto and say either add the text beneath all of the videos or remove them here